styles/planet-minecraft/catppuccin.user.css

```diff
@@ -128,6 +128,30 @@
       }
     }
 
+    .content-actions {
+          li {
+              background: @surface0 !important;
+              border-color: @surface0 !important;
+
+              a {
+                  color: @text !important;
+              }
+              
+              a:hover, .js_link:hover, .link:hover {
+                  background-color: @surface2 !important;
+                  color: @text !important;
+              }
+              
+              .material-icons {
+                  color: @subtext1 !important;
```

Comment:
This wasn't the original color, and you missed the part where if any of the links above are hovered the icon changes to accent.